examples/http-proxy/http_proxy.py

```diff
@@ -173,19 +176,15 @@ async def main(
 
         network = await golem.create_network("192.168.0.1/24")
         cluster = await golem.run_service(HttpService, network=network, num_instances=num_instances)
-
-        def instances():
-            return [f"{s.provider_name}: {s.state.value}" for s in cluster.instances]
+        instances = cluster.instances
```

Comment:
I still sustain the point that using `cluster.instances` below would more clearly show that we're using a dynamic property on a cluster and not some once frozen set of data... not everyone must know that `cluster.instances` is a dynamic property ... 

_especially_ also because even if you look at the code, you're not passing a reference but performing a _copy_ of it, so that someone might assume it's _supposed_ to be a snapshot ...